docker/Dockerfile

```diff
@@ -113,37 +84,46 @@ RUN mkdir /tmp/openmpi && \
     ldconfig && \
     rm -rf /tmp/openmpi
 
-# Install OpenSSH for MPI to communicate between containers
-RUN apt-get install -y --no-install-recommends openssh-client openssh-server && \
-    mkdir -p /var/run/sshd
+# Configure OpenMPI to run good defaults:
+#   --bind-to none --map-by slot --mca btl_tcp_if_exclude lo,docker0 --mca btl_vader_single_copy_mechanism none
+RUN echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params.conf && \
+    echo "rmaps_base_mapping_policy = slot" >> /usr/local/etc/openmpi-mca-params.conf && \
+    echo "btl_tcp_if_exclude = lo,docker0" >> /usr/local/etc/openmpi-mca-params.conf
 
 # Set env to install horovod with nccl and tensorflow option
-ENV HOROVOD_GPU_ALLREDUCE NCCL
-ENV HOROVOD_WITH_TENSORFLOW 1
+ENV HOROVOD_GPU_ALLREDUCE=NCCL \
+    HOROVOD_WITH_TENSORFLOW=1
```

Comment:
It looks good for me to put this ENV line before [this line](https://github.com/blue-oil/blueoil/pull/580/files#diff-ebacf6f6ae4ee68078bb16454b23247dR109) because this is an installation option of `horovod`.

Reply:
changed